src/Environment.php

```diff
@@ -509,10 +510,15 @@ public function resolveTemplate($names)
                 return $name;
             }
 
+            // Optimization: Avoid throwing an exception when it would be ignored anyway.
+            if (1 !== $count && !$this->getLoader()->exists($name)) {
```

Comment:
As we are checking the count here we might refactor the code to not catch the exception in case count === 1 (see below).

Reply:
You're right. I was thinking that we still need the catch to display a proper error message, but the code will do it even without try/catch.